templates/zerver/help/roles-and-permissions.md

```diff
@@ -9,24 +9,24 @@ There are several possible roles in a Zulip organization.
   and organization settings.  Cannot create or demote organization
   owners.
 
-* **Moderator**: Have the permissions of Full Members; additionally,
-  many "Organization permissions" settings allow Moderators to be
+* **Moderator**: Have the permissions of full members; additionally,
+  many "Organization permissions" settings allow moderators to be
   given additional privileges or do so by default.
 
 * **Member**: Has access to all public streams.  Member is the default
   role for most users.  [Some organization
   settings](/help/restrict-permissions-of-new-members) allow an
-  organization to restrict the permissions of **New Members**; Members
-  who do not have those restrictions are called **Full Members**.
+  organization to restrict the permissions of **new members**; Members
+  who do not have those restrictions are called **full members**.
```

Comment:
I'm not sure about changing the capitalization of roles here; I think it's net helpful to make these proper noun tokens.